Freenode's ircd-seven uses a custom banlist type for mutes (+q).
This patch makes bip handle +q the same way as +q.
Without it, a user wouldn't get the banlist replies for mutes.

diff --git a/src/irc.c b/src/irc.c
index 556bf97..a5cc039 100644
--- a/src/irc.c
+++ b/src/irc.c
@@ -944,7 +947,8 @@ static int irc_cli_mode(struct link_client *ic, struct line *line)
 
 	/* This is a wild guess and that sucks. */
 	if (!irc_line_elem_equals(line, 0, "MODE") ||
-			strchr(irc_line_elem(line, 2), 'b') == NULL)
+			(strchr(irc_line_elem(line, 2), 'b') == NULL &&
+                        strchr(irc_line_elem(line, 2), 'q') == NULL))
 		return OK_COPY;
 
 	++ic->who_count;
